### PR TITLE
ACD 26

### PR DESCRIPTION
**Pull request by AdamB25**
Category DTO and Category Entity is  created review and merge

### CODE REVIEW

File: src/main/java/com/cocoon/entity/Category.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "category")
+public class Category extends BaseEntity{
+
+    private String description;
+    private int companyId;
+    private boolean enabled;
+    private int createdBy;
```
Comment (khicchi) [inline src/main/java/com/cocoon/entity/Category.java:23]: BaseEntity de mevcut.

File: src/main/java/com/cocoon/entity/Category.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    private int updatedBy;
```
Comment (khicchi) [inline src/main/java/com/cocoon/entity/Category.java:25]: BaseEntity de mevcut.

File: src/main/java/com/cocoon/entity/Category.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    private LocalDateTime createdTime;
```
Comment (khicchi) [inline src/main/java/com/cocoon/entity/Category.java:27]: BaseEntity de mevcut.

File: src/main/java/com/cocoon/entity/Category.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    private LocalDateTime updatedTime;
```
Comment (khicchi) [inline src/main/java/com/cocoon/entity/Category.java:28]: BaseEntity de mevcut.

**Review by khicchi — CHANGES_REQUESTED** [review]

**Comment by AdamB25** [comment]
Ok

**Review by mmsezgin — APPROVED** [review]
Good job.

**Review by mmsezgin — APPROVED** [review]

**Review by khicchi — APPROVED** [review]

**Pull request merged**